client/src/components/GameInfoCarousel/styles.js

```diff
@@ -6,44 +6,55 @@ export const Container = styled.div`
   align-items: center;
   height: 100%;
   width: 100%;
-  background: url('https://store.cloudflare.steamstatic.com/public/images/v6/app/game_page_background_shadow.png?v=2');
-  background-position: bottom;
-  background-repeat: no-repeat;
   padding: 20px;
+  margin-top: 150px;
 `;
 
 export const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
-  width: 50%;
-  height: 50%;
+  align-items: center;
+`;
+
+export const BackgroundWaterMark = styled.div`
+  display: flex;
+  justify-content: center;
+  background: url('https://store.cloudflare.steamstatic.com/public/images/v6/app/game_page_background_shadow.png?v=2');
+  background-position: bottom;
+  background-repeat: no-repeat;
+  min-width: 885px;
+  max-width: 50%;
 `;
 
 export const MainGameInfoWrapper = styled.div`
   display: flex;
+  height: 50%;
+  width: 50%;
+  justify-content: center;
 
 ::before {
+    border-radius: 1px;
+    border-width: thick;
     content: "";
     background: url("${(props) => props.backgroundImage}");
     background-repeat: no-repeat;
     background-position: center;
-    background-size: 2000px 1500px;
-    border-radius: 5000px;
+    background-size: 1270px 715px;
     position: absolute;
     z-index: -1;
```

Comment:
Curious as to why you made z-index here -1? I've used it to set z-index to a positive number to give it priority over other elements on the page when it comes to display depth, maybe I'm just not familiar with setting it to -1 does? May look it up after this code review!

Reply:
yeah, I don't think that actually made an impact, was just experimenting with some things. I'm gonna have to clean up my styles and take out the stuff that isn't actually doing anything.